core/templates/dev/head/exploration_editor/preview_tab/preview_tab.html

```diff
@@ -6,19 +6,53 @@
     <conversation-skin></conversation-skin>
   </div>
 
-  <!-- Developers: set ng-show="true" to see a parameter debug log. -->
-  <div ng-show="false">
-    <h4><strong>Parameter values</strong></h4>
-    <table>
-      <tr ng-repeat="(paramName, paramValue) in allParams">
-        <td style="padding: 1px 5px;"><[paramName]></td>
-        <td style="padding: 1px 5px;"><[paramValue ? paramValue : null]></td>
-      </tr>
-    </table>
+  <div class="oppia-parameters">
+    <div class="oppia-parameter-summary">
```

Comment:
Can this whole div only show up if:
- explorationAdvancedFeaturesService.areParametersEnabled() is true, and
- there is at least one parameter?

Otherwise, it gets somewhat distracting...